### ISSUE TITLE
Build for the aarch64 platform

### ISSUE THREAD

**Issue by mathbunnyru** [conda-forge/r-tidymodels-feedstock#15]
### Comment:

This package depends on the `r-yardstick` package, and it is not built for aarch64.
So we first need to fix `r-yardstick`.
Here is the issue:
https://github.com/conda-forge/r-yardstick-feedstock/issues/13

**Comment by mathbunnyru** [comment]
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4080

**Comment by mfansler** [comment]
`r-dicedesign` is available. Next missing is `r-slider` (needed by `r-rsample`).

```bash
% CONDA_SUBDIR=linux-aarch64 mamba create -n foo r-tidymodels r-base=4.2

Looking for: ['r-tidymodels', 'r-base=4.2']

pkgs/main/linux-aarch64                                       No change
bioconda/linux-aarch64                                        No change
pkgs/r/linux-aarch64                               122.0 B @ 244.0 B/s  0.1s
pkgs/r/noarch                                                 No change
pkgs/main/noarch                                              No change
bioconda/noarch                                    @   3.3MB/s  1.4s
conda-forge/linux-aarch64                          @   3.7MB/s  2.4s
conda-forge/noarch                                 @   2.5MB/s  4.8s
Could not solve for environment specs
Encountered problems while solving:
  - nothing provides r-slider >=0.1.5 needed by r-rsample-0.0.9-r36hc72bb7e_0

The environment can't be solved, aborting the operation
```

**Comment by mathbunnyru** [comment]
Thanks, @mfansler. We're getting closer 😄

**Comment by mathbunnyru** [comment]
PR for `r-slider`: https://github.com/conda-forge/r-slider-feedstock/pull/15

**Comment by mathbunnyru** [comment]
And, it seems to work now 🎉

**Comment by mfansler** [comment]
@mathbunnyru for future reference, I just realized we could have made this process much simpler by using Mamba's `repoquery` tools. Specifically, if we would have run:

```bash
mamba repoquery depends -c conda-forge -p linux-aarch64 r-tidymodels
```

it would have given us *all* the missing dependencies and we could have submitted them all for migration in a single PR. I wrote up a current example for [a StackOverflow question](https://stackoverflow.com/q/76388968/570918) if you're interested.

That's all - hope you're well!
